src/app/components/header/Header.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Header from './Header';
+
+describe('Header', () => {
+  describe('Component', () => {
+    it('should render correctly', () => {
+      const tree = renderer.create(<Header />);
+      expect(tree).toMatchSnapshot();
+    });
+
+    // it('should have a font-size of x', () => {
```

Comment:
Did you mean to add tests here?

Reply:
Yes, I may remove them to Cypress for integration